### ISSUE TITLE
An NPC with the Cleric Sheet assigned shows "Undefined" for spell/ability rolls

### ISSUE THREAD

**Issue by algnc** [foundryvtt-dcc/foundryvtt-dcc#118]
If an NPC is created, and assigned the Cleric sheet, if you roll their abilities, you get undefined in the chat as the roll title. No console output is produced.

![image](https://user-images.githubusercontent.com/7275595/106369028-2c654300-631c-11eb-91b1-304bfdd2ec69.png)

**Comment by mooped** [comment]
The class sheets only work for 'Player' type actors that have a full set of stats as opposed to a statline (like monsters and NPCs).

There are probably improvements to be made with the naming as NPC implies any character that is controlled by the judge, when what we really mean is 'has a statline'.

Limiting the selection to prevent selecting a class sheet for an NPC, and maybe changing the behaviour of the current 'Zero/NPC' sheet (which selects one of two sheets based on actor type) could also help.

**Comment by mooped** [comment]
Tempted to rename the types to 'Sheet' and 'Statline' to make the distinction clearer, but I'm also messing with a new way of importing actors (creating pregens is currently a pain) that sidesteps the issue a lot of the time.

**Comment by mooped** [comment]
Any renaming of the actor types should happen before v1.0 to avoid issues (actors can be migrated to other types, but it's easier to not have to worry about it for new users).

**Comment by mooped** [comment]
Superseded by #140.
